homeassistant/components/zwave/__init__.py

```diff
@@ -542,6 +546,12 @@ def print_config_parameter(service):
         _LOGGER.info("Config parameter %s on Node %s : %s",
                      param, node_id, get_config_value(node, param))
 
+    def print_node(service):
+        """Print all information about z-wave node."""
+        node_id = service.data.get(const.ATTR_NODE_ID)
+        node = NETWORK.nodes[node_id]
+        nice_print_node(node)
```

Comment:
In debug mode the screen is spammed too much(because of the printout for all nodes)

This is useful for development and maybe for power-users - to understand why a node was added as such-and-such device, etc.

I think it is OK that it doesn't goes to the log file.